spec/fixtures/pom.xml

```diff
@@ -90,6 +90,12 @@
                 <artifactId>commons-lang</artifactId>
                 <version>2.6</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>${guava.version}</version>
+            </dependency>
```

Comment:
here I'm moving the `<version>` field for guava from `<dependencies>` into `<dependencyManagement>` to assert that we'll use the latter as fallback.